app/javascript/components/gtl/DataTable.jsx

```diff
@@ -51,6 +51,11 @@ export const DataTable = ({
              key={'selectAll'}
              checked={!!checkedItems['selectAll']}
              onChange={event => localOnSelectAll(rows, event)}
+             onKeyPress={event => localOnSelectAll(rows, event)}
+             role="checkbox"
+             aria-checked="false"
+             tabIndex="0"
+             aria-labelledby="chk1-label"
```

Comment:
Are you sure this is set correctly? According to the docs this should point to an [existing ID](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute)